modules/orchestrator/proc-install-components-for-orchestrator-plugin.adoc

```diff
@@ -15,7 +15,7 @@ To run the Orchestrator plugin successfully on {ocp-short}, you must install com
 +
 [NOTE]
 ====
-The {product} Operator provisions these components automatically. Alternatively, you can install them manually by using another Operator.
+You must preinstall the components, or install them by using one of the supported methods described in the following section (manual installation, helper script, or Helm chart).
```

Comment:
> one of the supported methods described in the following section (manual installation, helper script, or Helm chart).

🤔 So here, we are saying the helper script is a supported method, but at the same time, saying "Do not use it in production." ?

Reply:
That is what upstream doc says:
"This script provides a quick way to install the OpenShift Serverless infrastructure for the Orchestrator plugin. It is safe to use in empty clusters but should be used with caution in production clusters."

Reply:
@rm3l not sure I got your point, sorry. You mean the phrasing is not good (which might be the case) or we should not mention it here because it has to be applied with caution for prod (but is incomparable simpler than other) or what? :)